protos/sources/vega/data_source.proto

```diff
@@ -69,6 +69,14 @@ message EthCallSpec {
   repeated google.protobuf.Value args = 4;
   // Conditions for determining when to call the contract method.
   EthCallTrigger trigger = 5;
+  // Number of confirmations required before the query is considered verified
+  uint64 required_confirmations = 6;
+
+  // Below - is this common across all sources? so should be on DataSourceDefinitionExternal?
```

Comment:
Is this a comment directed at the core team?

Reply:
Yeah, at the others working on this feature, something we need to figure out, note this PR is to the feature branch, not to develop, this comment wouldn't make it to develop.